### DISCUSSION TITLE
Workout Timezone on Github

### DISCUSSION THREAD

**Discussion by crtvxdesign** [philosowaffle/peloton-to-garmin#303] [Q&A] [answered]
I have created a fork on Github and everything is working great except the time Garmin Connect is showing my workout. Where do I change my timezone in Github? Thank you

**Comment by philosowaffle** [comment, marked as answer]
You can add an env variable [here](https://github.com/philosowaffle/peloton-to-garmin/blob/master/.github/workflows/sync_peloton_to_garmin.yml#L84) to specify timezone information. Such as:

```yaml
   TZ: America/Chicago
```

**Reply by crtvxdesign** [reply]
Thank you.

**Comment by wesraynbirdtilbury** [comment]
Does `Europe/London' work?
